nuxeo-core/nuxeo-core-convert-api/src/main/java/org/nuxeo/ecm/core/convert/api/ConversionService.java

```diff
@@ -131,4 +147,11 @@ BlobHolder convertToMimeType(String destinationMimeType, BlobHolder blobHolder,
      * @since 7.4
      */
     BlobHolder getConversionResult(String id, boolean cleanTransientStoreEntry);
+
+    /**
+     * Adjusts the blob file name
+     *
+     * @since 9.2
+     */
+    void adjustBlobName(String filename, Blob blob, String extension, String mimeType);
```

Comment:
If it's just to be able to test it then please don't add it to the interface. The test can cast to `ConversionServiceImpl` if needed.